package.json

```diff
@@ -20,16 +20,9 @@
     "preset": "jest-expo"
   },
   "dependencies": {
-    "@splitsoftware/splitio": "^10.8.1",
-    "cryptiles": "^4.1.3",
-    "events": "^3.0.0",
-    "expo": "^33.0.7",
-    "extend": "^3.0.2",
-    "merge": "^1.2.1",
-    "randomatic": "^3.1.1",
-    "react": "16.8.3",
-    "react-native": "0.59.8",
-    "rox-react-native": "^2.3.7",
-    "util": "^0.11.1"
+    "@splitsoftware/splitio": "^10.8.4-canary.2",
+    "expo": "^34.0.1",
+    "react": "16.8.6",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz"
```

Comment:
Any particular reason to use the link to the .tar.gz instead of downloading from npm repository?

Reply:
This dependency was added by "expo init" command. Changing it to one using semantic versioning spec, generated a Warning in the expo UI.